BRANCHING_STRATEGY.md

```diff
@@ -0,0 +1,20 @@
+
+# Caliptra 2.0 Branching Strategy
+
+A new branch, named **"main-2.x"**, will be created off of the main branch for Caliptra 2.0 development. This includes changes to ROM, FMC and Runtime.
+
+- The **"main-2.x"** branch will not maintain backward compatibility with 1.x features.
+- The **"main-2.x"** branch will adhere to the same rules, policies, and CI pipelines as the main branch.
+- Any fixes made to the main (1.x) branch will be forward-ported to the **"main-2.x"** branch. These fixes will be cherry-picked from main to main-2.x to maintain a linear history. This will be a weekly exercise.
```

Comment:
We might want to define a little more clearly what "fixes" means. Is this all commits?

Also, who will be doing this exercise?

Reply:
This means all the commits relevant to 2.0. Some commits will be simple cherry picks (this commit) vs some will have to be done manually. I have volunteered to do this, but the more the merrier.